docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==4.5.0
+myst-parser==0.18.1
```

Comment:
👍  a nice place to put this file that for some reason never occurred to me before